pkgs/tools/admin/ssh-import-id/default.nix

```diff
@@ -0,0 +1,36 @@
+{ buildPythonPackage
+, stdenv
+, fetchgit
+, requests
+, makeWrapper
+, extraHandlers ? []
+}:
+
+buildPythonPackage rec {
+  pname = "ssh-import-id";
+  version = "5.8";
+
+  src = fetchgit {
+    url = "https://git.launchpad.net/ssh-import-id";
+    rev = version;
+    sha256 = "0l9gya1hyf2qfidlmvg2cgfils1fp9rn5r8sihwvx4qfsfp5yaak";
+  };
+
+  propagatedBuildInputs = [
+    requests
+  ];
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  # handlers require main bin, main bin requires handlers
+  makeWrapperArgs = [ "--prefix" ":" "$out/bin:${stdenv.lib.makeBinPath extraHandlers}" ];
```

Comment:
I actually realized you don't need that at all, wrap.sh adds all inputs to PATH in the wrapper https://github.com/NixOS/nixpkgs/blob/bee470ef9a8dc8af085e89135be94875552198f7/pkgs/development/interpreters/python/wrap.sh#L101. You can confirm this by looking at any python apps wrapper.

What you can instead do is just add `extraHandlers` to `propagatedBuildInputs`

Reply:
Ok, added. Still need makeWrapperArgs for $out/bin tho